### ISSUE TITLE
pip install fails when using sudo

### ISSUE THREAD

**Issue by kseistrup** [ondergetekende/python-panavatar#1]
I can easily install panavatar in a virtualenv:

``` sh
$ virtualenv venv
Using base prefix '/usr'
New python executable in /tmp/venv/bin/python3
Also creating executable in /tmp/venv/bin/python
Installing setuptools, pip, wheel...done.
$ source venv/bin/activate
(venv) $ pip install --upgrade panavatar
Collecting panavatar
  Downloading panavatar-0.3.0.tar.gz
Building wheels for collected packages: panavatar
  Running setup.py bdist_wheel for panavatar ... done
  Stored in directory: /home/kseistrup/.cache/pip/wheels/73/80/eb/6950e01b5ec045eea5498cb4beeaf943c6bac3a0a6069ca5d1
Successfully built panavatar
Installing collected packages: panavatar
Successfully installed panavatar-0.3.0
```

However, installing panavatar globally with sudo fails:

``` pytb
$ sudo pip install --upgrade panavatar
Collecting panavatar
  Downloading panavatar-0.3.0.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-uh2stpq4/panavatar/setup.py", line 7, in <module>
        ext_modules = cythonize("panavatar/_natives.pyx")
      File "/usr/lib/python3.5/site-packages/Cython/Build/Dependencies.py", line 796, in cythonize
        aliases=aliases)
      File "/usr/lib/python3.5/site-packages/Cython/Build/Dependencies.py", line 688, in create_extension_list
        for file in nonempty(sorted(extended_iglob(filepattern)), "'%s' doesn't match any files" % filepattern):
      File "/usr/lib/python3.5/site-packages/Cython/Build/Dependencies.py", line 107, in nonempty
        raise ValueError(error_msg)
    ValueError: 'panavatar/_natives.pyx' doesn't match any files

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-uh2stpq4/panavatar/
```

Python 3.5.1
Cython 0.24
Linux 4.5.4 (64bit)

This is ArchLinux, so python == python3 and pip == pip3.

**Comment by kvdveer** [comment]
Thanks for reporting!

I haven't been able to reproduce this on arch, but the python docker image (based on Debian) was showing the same symptoms.

Fixed in 0.3.2

**Comment by kseistrup** [comment]
That was fast, thanks!
